config/development.cfg

```diff
@@ -3,5 +3,12 @@ SECRET_KEY = 'bad_secret_key'
 TESTING = False
 ES_PORT = 9201
 ES_HOSTS = ["localhost"]
+REGULOME_ES_PORT = 9201
+REGULOME_ES_HOSTS = ["localhost"]
+REGION_SEARCH_ES_PORT = 9201
```

Comment:
Are we loading Regulome and Encoded into same index and searching over everything?

Reply:
These configs is for development only to serve as a template, locally you can change the ports.